app/domain/status/rewriting_status_update_dto.py

```
@@ -0,0 +1,6 @@
+from app.domain.data.competency_dto import Competency
```

Ruff (0.8.2)

`app.domain.data.competency_dto.Competency` imported but unused

Remove unused import: `app.domain.data.competency_dto.Competency`

(F401)